hc/front/tests/test_add_channel.py

```diff
@@ -38,4 +38,26 @@ def test_instructions_work(self):
             self.assertContains(r, "Integration Settings", status_code=200)
 
     ### Test that the team access works
+    def test_team_access_works(self):
+        url = "/integrations/add/"
+        form = {"kind": "email", "value": "bob@example.org"}
+
+        # Logging in as bob, not alice. Bob has team access so this
+        # should work.
+        self.client.login(username="bob@example.org", password="password")
+        r = self.client.post(url, form)
+
+        self.assertRedirects(r, "/integrations/")
+        assert Channel.objects.count() == 1
```

Comment:
You should use `self.assertEqual`. e.g `self.assertEqual(400, r.status_code)`. Note that it's common, and a best practice to start with the expected value followed by the returned value. https://stackoverflow.com/questions/9331259/unit-testing-why-is-the-expected-argument-always-first-in-equality-tests

Reply:
Okay, I going change that.